distributed/bokeh/scheduler.py

```diff
@@ -288,6 +288,7 @@ def __init__(self, scheduler, width=600, **kwargs):
                         x='nbytes-half', y='y',
                         width='nbytes', height=1,
                         color='nbytes-color')
+            nbytes.axis[0].ticker = BasicTicker(mantissas=[1,256,512], base=1024)
```

Comment:
from the docs:

```
    ''' Generate "nice" round ticks at any magnitude.

    Creates ticks that are "base" multiples of a set of given
    mantissas. For example, with ``base=10`` and
    ``mantissas=[1, 2, 5]``, the ticker will generate the sequence::

        ..., 0.1, 0.2, 0.5, 1, 2, 5, 10, 20, 50, 100, ...

    '''
```

With the base arg of 1024, it should nicely handle the kb -> mb -> gb transitions. The mantissas specify that 1*b, 0.25*b, 0.5*b steps are desired. (Let me know if i'm not explaining it well)